include/swift/AST/Expr.h

```diff
@@ -5428,7 +5339,7 @@ inline bool ApplyExpr::validateArg(Expr *e) const {
   else if (isa<BinaryExpr>(this))
     return isa<TupleExpr>(e);
   else
-    return isa<ParenExpr>(e) || isa<TupleExpr>(e) || isa<ArgumentShuffleExpr>(e);
+    return isa<ParenExpr>(e) || isa<TupleExpr>(e);
```

Comment:
I've been waiting for this day since I first added `ApplyExpr::validateArg()`. 🎉